psd-web/spec/features/sign_in_spec.rb

```diff
@@ -5,18 +5,22 @@
 
   let(:investigation) { create(:project) }
   let(:user) { create(:user, :activated, has_viewed_introduction: true) }
+  let(:password) { "2538fhdkvuULE36f" }
 
-  def fill_in_credentials
-    visit "/sign-in"
-
+  def fill_in_credentials(password_override: nil)
     fill_in "Email address", with: user.email
-    fill_in "Password", with: "2538fhdkvuULE36f"
+    if password_override
+      fill_in "Password", with: password_override
+    else
+      fill_in "Password", with: password
+    end
```

Comment:
Tiny suggestion (you may not like it so feel free to ignore it):
```suggestion
    fill_in "Password", with: password_override.presence || password
```